e2e/cypress/integration/onboarding/invalidate_pending_email_invitations_spec.js

```diff
@@ -0,0 +1,172 @@
+
+// Copyright (c) 2015-present Mattermost, Inc. All Rights Reserved.
+// See LICENSE.txt for license information.
+
+// ***************************************************************
+// - [#] indicates a test step (e.g. # Go to a page)
+// - [*] indicates an assertion (e.g. * Check the title)
+// - Use element ID when selecting an element. Create one if none.
+// ***************************************************************
+
+// Group: @onboarding
+
+import * as TIMEOUTS from '../../fixtures/timeouts';
+import {getEmailUrl, getEmailMessageSeparator, reUrl, getRandomId} from '../../utils';
+
+describe('Onboarding', () => {
+    let testTeam;
+    const usernameOne = `user${getRandomId()}`;
+    const usernameTwo = `user${getRandomId()}`;
+    const usernameThree = `user${getRandomId()}`;
+    const emailOne = `${usernameOne}@sample.mattermost.com`;
+    const emailTwo = `${usernameTwo}@sample.mattermost.com`;
+    const emailThree = `${usernameThree}@sample.mattermost.com`;
+    const password = 'passwd';
+
+    const baseUrl = Cypress.config('baseUrl');
+    const mailUrl = getEmailUrl(baseUrl);
+    let isLicensed;
+
+    before(() => {
+        // # If the instance the test is running on is licensed, assign true to isLicensed variable
+        cy.apiGetClientLicense().then(({license}) => {
+            isLicensed = license.IsLicensed === 'true';
+        });
+
+        // # Disable LDAP and do email test if setup properly
+        cy.apiUpdateConfig({LdapSettings: {Enable: false}});
+        cy.apiEmailTest();
+
+        cy.apiInitSetup().then(({team}) => {
+            testTeam = team;
+            cy.visit(`/${team.name}/channels/town-square`);
+        });
+    });
+
+    it('MM-T399 Invalidate Pending Email Invitations', () => {
+        // # Open the 'Invite People' full screen modal
+        cy.get('.sidebar-header-dropdown__icon').click();
+        cy.get('#invitePeople').find('button').eq(0).click();
+
+        // # Wait half a second to ensure that the modal has been fully loaded and check if the instance is licensed
+        cy.wait(TIMEOUTS.HALF_SEC);
+        checkIfLicensed();
+
+        // # Invite the first user (the one that won't be invalidated and logout)
+        inviteNewUser(emailOne);
+        cy.wait(TIMEOUTS.FIVE_SEC);
+        cy.apiLogout();
+
+        // # Get the email sent to the first user, verify the email and go to the provided link
+        getEmail(usernameOne, emailOne);
+
+        // # Type username and password
+        cy.get('#name').should('be.visible').type(usernameOne);
+        cy.get('#password').should('be.visible').type(password);
+
+        // # Attempt to create an account by clicking on the 'Create Account' button
+        cy.get('#createAccountButton').click();
+
+        // * Check that the display name of the team the user was invited to is being correctly displayed
+        cy.get('#headerTeamName').should('contain.text', testTeam.display_name);
```

Comment:
I'm getting random failing test here which can be solved by adding a timeout option.
```suggestion
        cy.get('#headerTeamName', {timeout: TIMEOUTS.HALF_MIN}).should('contain.text', testTeam.display_name);
```